src/Academic/Bundle/BugTrackingBundle/Controller/IssueController.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Academic\Bundle\BugTrackingBundle\Controller;
+
+use Academic\Bundle\BugTrackingBundle\Form\Handler\IssueHandler;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Academic\Bundle\BugTrackingBundle\Entity\Issue;
+use Academic\Bundle\BugTrackingBundle\Form\Type\IssueType;
+
+/**
+ * @Route("issue")
+ */
+class IssueController extends Controller
+{
+    /**
+     * @Route(
+     *      name="academic_bug_tracking_index",
+     * )
+     * @Template
+     */
+    public function indexAction()
```

Comment:
All methods with parameters should have @param docblock.
All methods that return something should have @return docblock.